source/reading.md

```diff
@@ -15,13 +15,9 @@ kernelspec:
 (reading)=
 # Reading in data locally and from the web
 
-We need to import the `pandas` package in order to read data into Python
-
-```{code-cell} ipython3
-import pandas as pd 
-```
 
 ```{code-cell} ipython3
+:tags: ["remove-cell"]
 import warnings
 warnings.filterwarnings('ignore')
```

Comment:
I think this may have been a hold-over from Navya's original version (which did spit out a lot of warnings). Will check to see if those are all resolved now...

Reply:
yup, there are no warnings. I will remove this.